lib/src/base-styles/_nx-text-input.scss

```diff
@@ -117,11 +117,25 @@
 
 .nx-text-input--textarea {
   .nx-text-input__box {
-    align-items: flex-end;
+    align-items: unset;
+    display: flex;
+    flex-direction: column;
+
+    &::before {
+      content: unset;
+    }
+
+    svg {
+      align-self: flex-end;
+      margin: var(--nx-spacing-2x) 0 0 0;
+    }
   }
 }
 
 textarea.nx-text-input__input {
+  //To disable horizontal resize of textarea
+  max-width: 100%;
+  min-width: 100%;
```

Comment:
Apparently there is a CSS `resize` property which I had somehow never heard of before now, which is a better way to accomplish this. https://developer.mozilla.org/en-US/docs/Web/CSS/resize

Reply:
> Apparently there is a CSS `resize` property which I had somehow never heard of before now, which is a better way to accomplish this. https://developer.mozilla.org/en-US/docs/Web/CSS/resize

I initially used `resize`, but didn't see it being used anywhere so went with an alternative approach.

changed to use `resize` here: c5741da20d55c5520c03381a3cfd9b5a4f966bfd